### ISSUE TITLE
Possible misuse of aria-label

### ISSUE THREAD

**Issue by abonnent** [validator/validator#1627]
Good afternoon. There is a lot of text in the a tag on my site, so pagespeed wrote about the discrepancy. Therefore, I put the aria-label in the tag where there is exactly the name that should be voiced for voicing for blind people

https://validator.w3.org/nu/?doc=https%3A%2F%2Fxn--d1acmfofh8i.xn--p1acf%2F - the page where the error is issued. 

if I'm wrong, then tell me how to do it better

my email: abonnent@inbox.ru

**Comment by stevefaulkner** [comment]
@abonnent from what i can see the text in the `div` and the `aria-label` are the same, in this case no `aria-label` is needed.
The reason you are getting the messages is that the div element maps to role=generic, which is not allowed to have an accessible name and will generally not be announced by screen reader software (refer to https://html5accessibility.com/stuff/2020/11/07/not-so-short-note-on-aria-label-usage-big-table-edition/ for more detail)

`<div class="title" itemprop="name" aria-label="Фёдор Бондарчук">Фёдор Бондарчук</div>`
